tests/v3_api/test_gke_cluster_provisioning.py

```diff
@@ -0,0 +1,129 @@
+from common import *   # NOQA
+
+
+CREDENTIALS = os.environ.get('GKE_CREDENTIAL', "")
+RANCHER_CLEANUP_CLUSTER = os.environ.get('RANCHER_CLEANUP_CLUSTER', "True")
+GKE_MASTER_VERSION = os.environ.get('GKE_MASTER_VERSION', "1.9.7-gke.1")
+
+
+def test_create_gke_cluster():
+
+    client = get_admin_client()
+    gkeConfig = get_gke_config()
+
+    print "Cluster creation"
+    cluster = client.create_cluster(gkeConfig)
+    clusterid = cluster.id
```

Comment:
@soumyalj What is the state you are waiting for here ? We already have intermediate_state check in validate_cluster method .

Reply:
As soon as the cluster is created, it temporarily goes to active state before it goes to provisioning state. Hence, I wait until it shows the correct state.

Reply:
Removed the wait